src/app/common/utils/common.utils.random.ts

```diff
@@ -3,5 +3,5 @@ export function getRandomNumber(min: number, max: number): number {
 }
 
 export function getRandomBoolean(): boolean {
-  return Math.random() < 0.5;
+  return Math.random() < 0.9;
```

Comment:
I understand why u did it -> error was happening too often on wallets page -> but now this function is "less random" than it was. Maybe you can make it configurable like:
```typescript
export function getRandomBoolean(randomFactor: number = 0.5): boolean {
  return Math.random() < randomFactor;
}
```
and then inside `fakeRequest` in `common.http.fake-request.ts` use it as `getRandom(0.9)`